src/config/app.config.ts

```diff
@@ -16,7 +16,7 @@ export default registerAs('app', () => ({
   host,
   port,
   uri,
-  env: process.env.EASEY_AUTH_API_ENV || 'local-dev',
+  env: process.env.EASEY_AUTH_API_ENV || 'development',
```

Comment:
default needs to stay local-dev so that it is differentiated from the actual dev environment